backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/view/DefaultView.java

```diff
@@ -357,6 +360,24 @@ public ViewResult getPage(
 			final int firstRow,
 			final int pageLength,
 			final ViewRowsOrderBy orderBy)
+	{
+		final PerformanceMonitoringService service = SpringContextHolder.instance.getBeanOr(
+				PerformanceMonitoringService.class,
+				NoopPerformanceMonitoringService.INSTANCE);
+		return service.monitor(
+				() -> getPage0(firstRow, pageLength, orderBy),
+				PerformanceMonitoringService.Metadata
```

Comment:
providing the `viewId` would be a valuable information.

Reply:
also, why do we have to measure here when we measure the ViewRestCOntroller.getViewData?

Reply:
in the dashboard you can see how long the underlying function inside a HTTP-Request take. This functions get handled together. Because of this they all must have the same Labels. If i add the 'viewId' here it would be ignored, because the other functions monitored in this Meter don't have a 'viewId' label.